sdk/azidentity/credential_log_classification.go

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azidentity
+
+import (
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+)
+
+const (
```

Comment:
Nit: Having this file for just the log classification const seems excessive.  Maybe move this to azidentity.go, or if there are to be (internal) logging helper funcs that go here maybe rename the file to something like logging.go?

Reply:
I'll move this and the log env vars func to a logging.go file. I think azidentity.go is a bit overloaded now.

Reply:
Done